composer.json

```diff
@@ -148,7 +148,12 @@
         "patchLevel": {
             "drupal/core": "-p2"
         },
-        "patches": {}
+        "patches": {
```

Comment:
@tannguyen04 
please find another way to not include these patches

Reply:
Hi @AlexSkrypnyk 
New release of search_api_solr is able to fix the issue. No need patches. 
https://www.drupal.org/project/search_api/issues/3454939#comment-15645298